src/views/textview/text-view-right.js

```diff
@@ -342,28 +334,14 @@ const rightSegmentContainer = (
     segmentNr,
     segText,
     number,
-    currentSegment,
-    currentPosition
   );
 };
 
 const rightSegment = (
   segmentNr,
   segText,
   number,
-  currentSegment,
-  currentPosition
 ) => {
-  if (
-    segmentNr == currentSegment &&
-    number > 10 &&
-    number < 180 &&
-    currentPosition < 100
-  ) {
-    // prettier-ignore
-    return html`<br /><span class="right-segment" id=${segmentNr} number="${number}">${segText}</span>`;
-  } else {
-    // prettier-ignore
```

Comment:
I think this prettier-ignore might need to stay.
We installed it because with the spans there were sometimes errors if it was prettified.